native/src/snappyclient/cpp/impl/ClientService.h

```diff
@@ -359,7 +363,14 @@ namespace impl {
         }
 
   };
+class NetConnection{
```

Comment:
NetConnection can be factored out in a different file as discussed.

Reply:
Factored out in different file